ts/src/kuna.ts

```diff
@@ -20,7 +22,7 @@ export default class kuna extends Exchange {
             'name': 'Kuna',
             'countries': [ 'UA' ],
             'rateLimit': 1000,
-            'version': 'v2',
+            'version': 'v4',
```

Comment:
@samgermain first build seems to be broken, second did you test this to make sure that the v4 methods work exactly as the v3 did?

Reply:
v4 will not work exactly as v3 (did you mean v2?). I believe that v2 and v3 are no longer available though because they are deprecated

Reply:
@samgermain but if they're not longer working, we should also update the unified methods, no?

Reply:
> @samgermain but if they're no longer working, we should also update the unified methods, no?

yeah, I'm making updates in stages to minimize the changes